### ISSUE TITLE
Extract consumer group member_count as a separate metric 

### ISSUE THREAD

**Issue by amuraru** [cloudhut/kminion#103]
Currently consumer group member_count is reported as a label in kminion_kafka_consumer_group_info metric.
When there is a high churn in a consumer group with members joining and leaving this leads to a high cardinality in this metric.
Moreover having a separate metric helps in charting the number of members in time.

**Comment by weeco** [comment]
I agree that this makes sense and I'm surprised I haven't done that from the beginning on. Thank you for raising this issue and also filing the PR. I will try to review the PRs in the course of next week. Feel free to ping me in case it takes longer

**Comment by amuraru** [comment]
Thanks @weeco - sure, take your time.
similiar scalar labels exists in topic_info metric: partition_count and replication_factor (if enabled) albeit more stable. Wondering if we should do the same there

**Comment by weeco** [comment]
Technically you are right about the two other metrics, but I think given the stability it's fine to keep them as label as part of the topic info metric.

**Comment by amuraru** [comment]
Another disadvantage I've see with these numerical labels is when writing alert rules.  I couldn't find a way create a prometheus rule that can check say: alert if number of any topic partitions go beyond 1000
Dashboarding is fine though as Grafana provides a way to chart extract and display label values
